lib/plugins/acts_as_customizable/lib/acts_as_customizable.rb

```diff
@@ -155,7 +155,7 @@ def respond_to?(method, include_private = false)
         def for_custom_field_accessor(method_symbol)
           match = /\Acustom_field_(?<id>\d+)=?\z/.match(method_symbol.to_s)
           if match
-            custom_field = CustomField.find_by_id(match[:id])
+            custom_field = available_custom_fields.find { |cf| cf.id.to_s == match[:id] }
```

Comment:
Is that your own idea @houndci ?

I asked Cpt. Rubocop and he does not complain. I will cite his assessment here:

> 1 file inspected, no offenses detected